qiita_pet/templates/user_messages.html

```diff
@@ -0,0 +1,10 @@
+{% extends sitebase.html %}
+{% autoescape None %}
+{% block content%}
+<h2 style="text-decoration: underline;">System Messages</h2>
+<ul>
```

Comment:
Uhm... that's not standard procedure across the code base... Unsure if it can generate problems in the future if we decide to add more functionality to this page. My vote goes to better be safe than sorry, so please change to follow what is more widely used in the system.